docs/v1/P4Runtime-Spec.mdk

```diff
@@ -5680,6 +5693,8 @@ man-in-the-middle attacks between the server and client.
 * Add new `OPTIONAL` match kind. At the moment, `OPTIONAL` is only supported by
   the v1model architecture [@v1model], and not by PSA. We hope that it will be
   included in the core P4 language by the time P4Runtime v1.2.0 is released.
+* Added the ability to change the ID of table match fields, action parameters,
+  and metadata for packetio in P4Info by using the `@id` annotation.
```

Comment:
```suggestion
  and Packet IO metadata fields in P4Info by using the `@id` annotation.
```
for consistency with the text in the section below

Reply:
Done